ext/opencensus.c

```diff
@@ -46,6 +46,7 @@ ZEND_END_ARG_INFO()
 ZEND_BEGIN_ARG_INFO_EX(arginfo_opencensus_trace_set_context, 0, 0, 1)
     ZEND_ARG_TYPE_INFO(0, traceId, IS_STRING, 0)
     ZEND_ARG_TYPE_INFO(0, parentSpanId, IS_STRING, 1)
+    ZEND_ARG_ARRAY_INFO(0, baggage, 1)
```

Comment:
Why pass by value?

Reply:
then you have to deref it and do other things since you are taking reference as zval from calling php script.